test/qtapp/build/.gitignore

```diff
@@ -0,0 +1,2 @@
+*
+!.gitignore
```

Comment:
`build` folder as well as `.user` files shall be .gitignored:
https://github.com/github/gitignore/blob/master/CMake.gitignore
https://github.com/github/gitignore/blob/master/C%2B%2B.gitignore